### ISSUE TITLE
training scales

### ISSUE THREAD

**Issue by cthacker** [experiencor/keras-yolo3#42]
With the grid scales == [ 1, 1, 1]  I notice that the losses obviously scale based on the grid size. I have a few questions related to this. 

1) Do you want the losses to be roughly equal per output head and if so is it fine to adjust the grid_scale accordingly ( ie [1, .5, .25])

2) What do you find is a good loss now for a model? It used to be that I knew I had a good yolov2 model when the loss was < 0.1. Now I am getting losses around ~20. Possible there are other issues at play here and I have made a mistake somewhere. I am not using your code as is, but instead incorporating the new loss in my own code.

**Comment by cthacker** [comment]
@experiencor Any insight into this from your experiences?

**Comment by experiencor** [comment]
1. nope, I adjust the grid_scale such that all 3 scales have good avg_obj
2. the loss in v3 is not divided by grid sizes; that's why it's much bigger; you should look at the printout to monitor the performance of the network.
